.github/workflows/maven-publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish snapshots to maven
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'main'
+      - '1.*'
```

Comment:
It won't work for 1.x branch since the `publishPluginZipPublicationToSnapshotsRepository` is only present starting 2.x?